conda/python-interface/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "python-interface" %}
+{% set version = "1.4.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "9516c60d014549238b6c983c598ffba89d93fc8556bdb5dc761915dd92b05b72" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - typing >=3.5.2
+    - funcsigs >=1.0.2    [py<30]
```

Comment:
Why are these required for the build (as opposed to run)?

Reply:
I'm not too sure about this one. Conda automatically added this into the recipe file for both build and run, so I trusted its judgement.